boards/arm/rpi_pico/rpi_pico_defconfig

```diff
@@ -7,3 +7,4 @@ CONFIG_UART_CONSOLE=y
 CONFIG_GPIO=y
 CONFIG_USE_DT_CODE_PARTITION=y
 CONFIG_BUILD_OUTPUT_UF2=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
this is updating board: should go into a separate commit

Reply:
Can be in the same commit with dts modifications?

Reply:
Done!

Reply:
nit: for bisectability this commit needs to go after adding interrupt support

Reply:
Okay, makes sense.